### PR TITLE
PR to dev branch.

### PR DESCRIPTION
**Pull request by talueses**
It is sent first to "dev" branch to evaluate and to later do the merge with "master" branch.

### CODE REVIEW

File: .eslintrc.js [hunk 1]
```diff
@@ -0,0 +1,17 @@
+module.exports = {
+  root: true,
+  env: {
+    node: true,
+  },
+  extends: [
+    'plugin:vue/essential',
+    '@vue/airbnb',
```
Comment (JoshDev1205) [inline .eslintrc.js:8]: Can we use Standard  ???? :pizza:

Comment (talueses) [reply .eslintrc.js:8]: Show me the code. Do you mean without "airbnb" linter?

Comment (JoshDev1205) [reply .eslintrc.js:8]: Change airbnb in favor or standardjs :100: 

https://standardjs.com/

**Comment by talueses** [comment]
Check @JoshDev1205 I change to "standard".

**Comment by talueses** [comment]
@JoshDev1205 ¿le echaste un ojo? Haces el PR y lo mandas al repositorio nuevo.

**Pull request merged**